app/bio/page.tsx

```diff
@@ -1,28 +1,78 @@
   return (
     <main>
-      <h2>Bio</h2>
-      <div style={{ display: "flex", flexDirection: "row", gap: "16px" }}>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyItems: "center",
+          gap: "16px",
+        }}
+      >
         <div>
-          <Image src={biopic} alt="biopic" width="800" height="300" />
+          <Image
+            src={biopic}
+            alt="biopic"
+            width="1600"
+            height="1200"
+            className="rounded-full aspect-square object-cover"
+          />
         </div>
-        <div>
-          <p>
-            Lorem ipsum odor amet, consectetuer adipiscing elit. Amet quam sem
-            ante dui in, facilisis nisi sit non. Cursus vulputate vel donec
-            ultrices aliquet condimentum quam varius. Leo vehicula nisi
-            tincidunt class lobortis placerat curabitur maecenas. Dignissim
-            torquent tempor penatibus fermentum ultricies interdum leo auctor.
-            Libero elit interdum vitae varius; ornare laoreet fringilla
-            adipiscing.
+        <div className="flex flex-col gap-2">
+          <p className={montserrat.className}>
+            I have been a correction officer, working in NY State and county
+            prisons for 20 years. 14 years ago, desperate to find a healthy
+            outlet and relieve the stresses of my job, I turned to yoga. I had
+            always heard that yoga was good for stress. I decided to try a
+            class, and instantly fell in love. This is what my body and mind had
+            been looking for. I started with Hot Yoga, and 6 months later, tried
+            all different styles of yoga. I felt at peace and blissful every
+            time I was on my mat. I soon realized that yoga was much more than
+            physical flexibility and strength. It was also about building mental
+            flexibility and strength. I noticed a huge difference, not just in
+            my body, but in my perceptions, attitudes, and thoughts, which
+            influenced my actions. I decided that I needed to share all the
+            benefits of yoga with as many people as I could. It's been my
+            passion and my dream to open up a yoga studio. I began doing Yoga
+            Teacher trainings, to further my own practice, and to expand my
+            knowledge of the many benefits of various styles of yoga.{" "}
+            <blockquote className={montserrat.className}>
+              {" "}
+              <p className="tab">
+                My goals are to help people understand that:{" "}
+                <blockquote className="indent-28">
+                  {" "}
+                  1. EVERYONE AND ANYONE can do yoga
+                </blockquote>{" "}
+                <blockquote className="indent-28">
+                  2. The MANY benefits of yoga
+                </blockquote>
+              </p>
+            </blockquote>
```

Comment:
You never want to nest `blockquote` elements: [MDN | Blockquote](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)

You also don't need to manually number ordered lists, just use `ol` tags in combination with `li`.

```suggestion
            <blockquote className={montserrat.className}>
              <p>My goals are to help people understand that:</p>
                 <ol>
                   <li>EVERYONE AND ANYONE can do yoga</li>
                   <li>The MANY benefits of yoga</li>
                 </ol>
            </blockquote>
```

Reply:
Edited and committed